source/administration/object-management/object-versioning.rst

```diff
@@ -18,7 +18,7 @@ Bucket Versioning
 Overview
 --------
 
-MinIO supports keeping multiple "versions" of an object in a single bucket.
+MinIO supports keeping up to 10,000 "versions" of an object in a single bucket.
```

Comment:
This section feels a bit odd - I understand why the flow is what it is, but it feels repetitive.

Maybe drop line 35 and have the `versionchanged` just on line 37-38?

Reply:
Reworked this section to avoid the repetition.